### PR TITLE
Problem: reward don't have stable p.a. rate

### PR DESCRIPTION
**Pull request by yihuang**
Solution: added a power function to adjust the rate based on `period_seconds / year_seconds`

### CODE REVIEW

File: docs/getting-started/staking.md [hunk 1]
```diff
@@ -94,16 +94,31 @@ The recording of block proposer is done in `BeginBlock` right before rewards dis
 
 ### Monetary expansion
 
-Monetary expansion is designed to release tokens from the reserve account to the reward pool, while keeping a fixed max total supply. The number of tokens being released at each period is defined as:
+Monetary expansion is designed to release tokens from the reserve account to the reward pool, while keeping a fixed max total supply.
 
+There are several configirations for this:
```
Comment (lezzokafka) [inline docs/getting-started/staking.md]: Can note that these are `rewards_config` in the genesis.

File: docs/getting-started/staking.md [hunk 2]
```diff
@@ -94,16 +94,31 @@ The recording of block proposer is done in `BeginBlock` right before rewards dis
...
+N = S * (pow(1 + R, P / Y) - 1)
+
+N: released coins
 S: total stakings at current block when reward distribution happens
-R_0: upper bound for the reward rate p.a. 
+R0: upper bound for the reward rate p.a.
 ```
 
-Note that this is an exponential decay function, where the index of  controls the “steepness” of the curve. Precisely, this damping factor controls the exponential decay rate of the reward function.
+Note that this is an exponential decay function, where the index of it controls the “steepness” of the curve. Precisely, this damping factor controls the exponential decay rate of the reward function.
```
Comment (lezzokafka) [inline docs/getting-started/staking.md:139]: Should we also cover ` "monetary_expansion_decay"` somewhere?

Comment (yihuang) [reply docs/getting-started/staking.md:139]: Yeah, I added it into the decay formular.

File: docs/getting-started/staking.md (hunk 1 above)
Comment (yihuang) [reply docs/getting-started/staking.md]: Done

**Review by lezzokafka — APPROVED** [review]

**Pull request merged**